### PR TITLE
Minor update to readme and coding style

### PR DESCRIPTION
**Pull request by artokin**
-Follow Mbed OS coding style
-Minor update to links in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,14 +26,15 @@ Example: `mbed compile -m K64F -t ARM`
...
 You can find more information on the network socket API in the [Mbed OS 5 documentation](https://docs.mbed.com/docs/mbed-os-api-reference/en/latest/APIs/communication/network_sockets/).
 
-You can also find more information on the socket statistic API in the [Mbed OS 5 documentation](https://docs.mbed.com/docs/mbed-os-api-reference/en/latest/).
+You can also find more information on the socket statistic API in the [Mbed OS 5 documentation](https://os.mbed.com/docs/latest/apis/socketstats.html).
 
  
 ### Note
 
-The current example is limited to the ethernet interface on supported devices. To use the example with a different interface, please follow the [socket example readme](https://github.com/ARMmbed/mbed-os-example-sockets/blob/master/README.md).
+The current example is configured to use the ethernet interface on supported devices. To use the example with a different interface, please follow the [socket example readme](https://github.com/ARMmbed/mbed-os-example-sockets/blob/master/README.md).
```
Comment (melwee01) [inline README.md]: Ethernet (capitalized)

File: README.md [hunk 2]
```diff
@@ -26,14 +26,15 @@ Example: `mbed compile -m K64F -t ARM`
...
  
 ### Note
 
-The current example is limited to the ethernet interface on supported devices. To use the example with a different interface, please follow the [socket example readme](https://github.com/ARMmbed/mbed-os-example-sockets/blob/master/README.md).
+The current example is configured to use the ethernet interface on supported devices. To use the example with a different interface, please follow the [socket example readme](https://github.com/ARMmbed/mbed-os-example-sockets/blob/master/README.md). 
+Network interfaces are documented in [Mbed OS 5 documentation](https://os.mbed.com/docs/latest/apis/network-interfaces.html)
 
 ### License and contributions
-The software is provided under Apache-2.0 license. Contributions to this project are accepted under the same license. Please see contributing.md for more info.
+The software is provided under Apache-2.0 license. Contributions to this project are accepted under the same license. Please see [Contrbuting instructions](CONTRIBUTING.md) for more information.
```
Comment (melwee01) [inline README.md]: *Contributing

Comment (melwee01) [reply README.md]: Contributing is missing an -i

**Review by melwee01 — APPROVED** [review]

**Comment by artokin** [comment]
@deepikabhavnani , would you please review?

**Review by deepikabhavnani — APPROVED** [review]
Thanks @artokin. Looks good to me 👍

**Pull request merged**